### PR TITLE
new estimator_states msg (split estimator_status)

### PR DESCRIPTION
**Pull request by dagar**
I'm looking to keep the default logging rate to a minimum. The current `estimator_status` message is relatively enormous and very often useful for flight review, but we don't need all the states (most available in other topics) and covariances for that high level overview.

I'd also like to be more realistic with the default logging rates for all of this large estimator topics.
 - estimator_innovation_test_ratios
 - estimator_innovation_variances
 - estimator_innovations
 - estimator_states
 - estimator_status

### CODE REVIEW

File: src/modules/logger/logged_topics.cpp [hunk 1]
```diff
@@ -57,10 +57,11 @@ void LoggedTopics::add_default_topics()
...
 	add_topic("ekf_gps_drift");
 	add_topic("esc_status", 250);
-	add_topic("estimator_innovation_test_ratios", 200);
-	add_topic("estimator_innovation_variances", 200);
-	add_topic("estimator_innovations", 200);
+	add_topic("estimator_innovation_test_ratios", 500);
+	add_topic("estimator_innovation_variances", 500);
+	add_topic("estimator_innovations", 500);
 	add_topic("estimator_sensor_bias", 1000);
+	add_topic("estimator_status", 500);
```
Comment (bresch) [inline src/modules/logger/logged_topics.cpp:64]:
```suggestion
	add_topic("estimator_states", 500);
```
:laughing:

File: src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp [hunk 2]
```diff
@@ -259,3 +226,50 @@ bool PreFlightCheck::ekf2Check(orb_advert_t *mavlink_log_pub, vehicle_status_s &
...
+	uORB::Subscription states_sub{ORB_ID(estimator_states)};
+	estimator_states_s states;
+
+	if (states_sub.copy(&states)) {
+
+		// check accelerometer delta velocity bias estimates
+		float test_limit = 1.0f; // pass limit re-used for each test
+		param_get(param_find("COM_ARM_EKF_AB"), &test_limit);
+
+		for (uint8_t index = 13; index < 16; index++) {
```
Comment (dagar) [inline src/modules/commander/Arming/PreFlightCheck/checks/ekf2Check.cpp:242]: Note for later, we could update this to use `estimator_sensor_bias` instead.

**Comment by dagar** [comment]
Merge conflict resolved.

**Review by bresch — APPROVED** [review]
Can we merge this?

**Comment by dagar** [comment]
> Can we merge this?

It was rolled into a multi-ekf support PR that's been merged.

**Pull request closed without merging**